src/main/java/teammates/ui/controller/InstructorEditInstructorFeedbackPageAction.java

```diff
@@ -34,15 +34,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                                 
         String moderatedQuestionId = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_MODERATED_QUESTION_ID);
 
-        Assumption.assertNotNull(String.format(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE,
-                                               Const.ParamsNames.COURSE_ID),
-                                 courseId);
-        Assumption.assertNotNull(String.format(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE,
-                                               Const.ParamsNames.FEEDBACK_SESSION_NAME),
-                                 feedbackSessionName);
-        Assumption.assertNotNull(String.format(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE,
-                                               Const.ParamsNames.FEEDBACK_SESSION_MODERATED_PERSON),
-                                 instructorUnderModerationEmail);
+        Assumption.assertNotNull(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, courseId);
```

Comment:
So we are going to find the NPE by looking at the stack trace instead of looking at the error message. XD